js/controller.js

```diff
@@ -498,6 +503,26 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
         }
         this.checkDeviceOrientation = false;
       }
+
+      // for Ads
+       
+      // mobile device detected
+      this.state.deviceOrientation.allow = true;
+      // current position of device
+      if(!this.state.deviceOrientation.freeze) {
```

Comment:
Can we add spaces between `if(`? Same comment for occurrences below.

Reply:
\+